README-PROVIDERS.md

```diff
@@ -83,11 +81,9 @@ automatically loaded, the null provider can be loaded instead.
 This can be useful if you are using non-default library contexts and want
 to ensure that the default library context is never used unintentionally.
 
-
 Loading Providers
 =================
 
-
```

Comment:
@levitte - is there any reason for these '-' ?

Reply:
Are you asking about dashes (used to make bullet lists, at the top of the file for a table of contents) or misreading my extra blank line deletion, or ask if they were there for a reason?

Reply:
Was there a reason.

Reply:
@slontis please look again, this is just a deletion of an extra blank line. IMO a correct change.